packages/types/src/envelope.ts

```diff
@@ -56,7 +56,7 @@ export type SessionItem =
   | BaseEnvelopeItem<SessionAggregatesItemHeaders, SessionAggregates>;
 export type ClientReportItem = BaseEnvelopeItem<ClientReportItemHeaders, ClientReport>;
 
-export type EventEnvelopeHeaders = { event_id: string; sent_at: string; baggage?: string };
+export type EventEnvelopeHeaders = { event_id: string; sent_at: string; trace?: Record<string, unknown> };
```

Comment:
I first thought, "why not use the `BaggageObj` type here" but then I realized that we can't because we declared the baggage types in @sentry/utils. For my PR I decided to move them (i.e. `Baggage` and `BaggageObj`) to @sentry/types and export it from there because I needed them in the types package. WDYT is this a good idea? If yes, we can either move them already in this PR or make the change afterwards. If not, I'll have to make some adjustments in my PR 😅

Reply:
Yes, we can move the types, good idea.